src/core/time.d

```diff
@@ -496,6 +496,77 @@ assert(std.datetime.Date(2010, 9, 7) - std.datetime.Date(2010, 10, 3) ==
  +/
 struct Duration
 {
+    /++
+        Converts this `Duration` to a `string`.
+
+        The string is meant to be human readable, not machine parseable (e.g.
+        whether there is an `'s'` on the end of the unit name usually depends on
+        whether it's plural or not, and empty units are not included unless the
+        Duration is `zero`). Any code needing a specific string format should
+        use `total` or `split` to get the units needed to create the desired
+        string format and create the string itself.
+
+        The format returned by toString may or may not change in the future.
+      +/
+    void toString (DGT) (scope DGT sink) const scope
+    {
+        alias SinkT = typeof(sink);
```

Comment:
Is `SinkT` equivalent to `DGT`?

Reply:
It is. Original code wasn't templated. Fix: https://github.com/dlang/druntime/pull/3756